org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/Results.java

```diff
@@ -0,0 +1,243 @@
+/*******************************************************************************
+* Copyright (c) 2017 Microsoft Corporation and others.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v10.html
+*
+* Contributors:
+*     Microsoft Corporation - initial API and implementation
+*******************************************************************************/
+
+package org.eclipse.jdt.ls.debug.adapter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.gson.JsonObject;
+
+/**
+ * The response content types defined by VSCode Debug Protocol.
+ */
+public class Results {
+    /**
+     * This monomorphic class is used to return results from a debugger request
+     * or to return errors. In addition events can be attached that are fired
+     * after the request results have been returned to the caller.
+     */
+    public static class DebugResult {
```

Comment:
Seems this is not necessary. DispatcherReponse/DispatcherEvent can be used directly.